src/DynamoCoreWpf/ViewModels/Preview/WatchViewModel.cs

```diff
@@ -139,9 +151,49 @@ public bool IsOneRowContent
             }
         }
 
+
+        /// <summary>
+        /// Number of items in the overall list if node output is a list
+        /// </summary>
+        public int NumberOfItemsWatchViewModel
```

Comment:
If `NumberOfItems` won't cause any issue, then I'd suggest to change to this name.